pkg/proxy/iptables/proxier.go

```diff
@@ -1099,6 +1126,52 @@ func (proxier *Proxier) syncProxyRules() {
 			activeNATChains[svcXlbChain] = true
 		}
 
+		if proxier.localDetector.IsImplemented() && svcChain != "" {
+			// We have a LocalDetector, so fill in the rules in the SVC chain to
+			// redirect external traffic (or simply masquerade it if we don't
+			// have a separate external chain).
+			var externalTrafficDest utiliptables.Chain
+			if svcXlbChain != "" {
+				externalTrafficDest = svcXlbChain
```

Comment:
This confuses me because it's not clear these are both going to return back to svcChain.  Is that right?